index.css

```diff
@@ -1,6 +1,7 @@
 body {
   width: 375px;
   margin: 0;
+  padding: 0;
```

Comment:
- [ ] Good job so far, but the changes you have added did not fix at it all, please consider making the requirement as mentioned in the previous

<img width="338" alt="pv" src="https://user-images.githubusercontent.com/77843050/206051449-3629ca41-e8f0-487a-862b-7110675020ea.png">